src/solver/variable/storage/average.h

```diff
@@ -71,15 +71,6 @@ namespace AllYears
 		Average()
```

Comment:
Un peu hors sujet mais les méthodes 
`void mergeHourInYear(uint year, uint hour, const double value)`
dans chacune des classes Average, StdDeviation, Min, Max, And, Raw, Empty, ...
sont inutiles

Reply:
A discuter, est ce que l'on crée une issue spécifique pour ce genre de situation ?

Reply:
après discussion, code supprimé